### DISCUSSION TITLE
Notification Appearance

### DISCUSSION THREAD

**Discussion by Auscobe** [valinet/ExplorerPatcher#2193] [Ideas]
It would be quite nice to be able to change where notifications will appear, in terms of corner (or not even just corner, but somewhere else, sides?), screen, perhaps even depending on if particular apps, programs, or a game is running. For example, on a three monitor setup, have notifications appear on the bottom left corner of the right screen, instead of the bottom right of the middle screen.

~~Another thing that would be nice is, especially if this is implemented, force notifications to show regardless of "focus assist" settings and whatnot, because I don't know how my Windows is still not showing me notifications during games ever despite that I have that configured. I have it on Priority only with the phone app set to it and it don't work. (If this is just a problem on my end, great, I'd appreciate help fixing it if possible, also perhaps adding apps not normally allowed to Focus assist? Though not sure that's within the scope of this.)~~ This was just me not knowing how Windows (default settings) works.

Thank you in advance, whatever happens.

**Comment by terax6669** [comment]
I guess this is duplicate of #779

**Reply by Auscobe** [reply]
Okay, is there anything else I might be able to use for this then?

**Reply by pyrates999** [reply]
I thought I did find a github repo that allows moving of the notifications, but that no longer exists: https://www.reddit.com/r/windows/comments/tyfg2n/windows_notifications_at_the_top_of_the_screen/

There's this, but it doesn't include a setting for moving the location of notifications: https://www.xda-developers.com/how-to-customize-notification-settings-windows-11/
